lambdas/preview/index.py

```diff
@@ -32,28 +32,14 @@
     'additionalProperties': False
 }
 
-Draft4Validator.check_schema(SCHEMA)
-VALIDATOR = Draft4Validator(SCHEMA)
-
-def lambda_handler(event, _):
+@api(cors_origins=ALLOWED_ORIGINS)
+@validate(SCHEMA)
```

Comment:
this should really be a JSON file; and `test_validator` should use the same file, instead of a made-up `dict`

Reply:
So pass a file path instead?

Reply:
yeah

Reply:
So I moved things around, and not sure this still makes sense. The validator tests are now in shared, but so the preview schema doesn't make sense there. The preview schema does get tested in `test_bad`; if that's not enough, I can add more things there.

Reply:
this is OK for now. but, and i only just realized this, schema's should be stand-alone files so that they can be checked into catalog, verified by javascript, etc.